composer/algorithms/progressive_resizing/progressive_resizing.py

```diff
@@ -181,3 +202,50 @@ def apply(self, event: Event, state: State, logger: Optional[Logger] = None) ->
                                              mode=self.mode,
                                              resize_targets=self.resize_targets)
         state.batch = (new_input, new_target)
+
+        if logger is not None:
+            logger.metric_batch({
+                "progressive_resizing/height": new_input.shape[2],
+                "progressive_resizing/width": new_input.shape[3],
+                "progressive_resizing/scale_factor": scale_factor
+            })
+
+
+def _make_crop(tensor: torch.Tensor, scale_factor: float) -> Callable[[torch.Tensor], torch.Tensor]:
```

Comment:
Might be nicer to add a
```
T_ResizeTransform = Callable[[torch.Tensor], torch.Tensor]
```
to simplify the return types for these helper functions

Reply:
Done